lib/Main.js

```diff
@@ -409,7 +410,7 @@ Main.prototype.onMatrixEvent = function(ev) {
     if (ev.type === "m.room.member"
         && this._bridge.getBot()._isRemoteUser(ev.state_key)
         && ev.sender !== myUserId) {
-        console.log(`${ev.state_key} got invite for ${ev.room_id} but we can't do DMs, warning room.`);
+        log.debug(`${ev.state_key} got invite for ${ev.room_id} but we can't do DMs, warning room.`);
```

Comment:
warn or info? Seems like something that is a little less debug-y

Reply:
It only happens if you try to PM the bridge, but it's rare enough for an info I guess.